02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -178,36 +178,49 @@ The [HGNC Comparison of Orthology Predictions (HCOP)](https://www.genenames.org/
 
-First, we need to download the file from the server holding the HGNC data.
-Go to this [directory page of the HGNC Comparison of Orthology Predictions (HCOP) files](ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/).
+We can automatically download the human zebrafish file we need for this example using `download.file()` command. 
+For this notebook, we want to download the file named `human_zebrafish_hcop_fifteen_column.txt.gz`.
 
-This is where the files that reflect the data provided via the [HGNC database](https://www.genenames.org/) are maintained.
-Ortholog species files with the '6 Column' output returns the raw assertions, Ensembl gene IDs and Entrez Gene IDs for human and one other species, while the '15 Column' output includes additional information such as the chromosomal location, accession numbers and the databases that support the assertions.
+First we'll declare a sensible file path for this. 
 
-*Note:* If you are using Safari (or the above FTP server link does not open in a web browser), you may need to go to the [link for the HCOP search tool](https://www.genenames.org/tools/hcop/) and scroll down to "Bulk Downloads" to choose a file to download.
-Here, you can find the same files you would find at the server linked above.
+```{r}
+# Declare what we want this file to be called once we download it
+zebrafish_hgnc_file <- file.path(
+  data_dir,
+  # Can change what the file will be called locally
+  "human_zebrafish_hcop_fifteen_column.txt.gz"
+)
+```
 
-To download a file, click the file name.
-For this notebook, you will want to download the file named `human_zebrafish_hcop_fifteen_column.txt.gz`.
-If you are using a different dataset, you can replace `zebrafish` in `human_zebrafish_hcop_fifteen_column.txt.gz` with the name of the species you have data for, and click on that file to download.
+Using the file path we just declared, we can use the `destfile` argument to download the file to this directory and use this file name. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/07499927a22ed7aa045245e40d692f8e49d7ba52/template/screenshots/download-hgnc-zebrafish-file.png" width=400>
+```{r}
+download.file(
+  paste0(
+    "ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/",
+    # Can replace with the file name of the species conversion you are looking for
+    "human_zebrafish_hcop_fifteen_column.txt.gz"
+  ),
+  # The file will be saved to this location and with this name
+  destfile = zebrafish_hgnc_file
+)
+```
 
-Next, move the `human_zebrafish_hcop_fifteen_column.txt.gz` file into your `data/` folder.
+If you are looking for a different species, see the [directory page of the HGNC Comparison of Orthology Predictions (HCOP) files](ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/) and find the file name of the species you are looking for
+If you are using Safari as your web browser see the directory using [this link](http://ftp.ebi.ac.uk/pub/databases/genenames/hcop/).
```

Comment:
The `http` link should work in any browser, so let's just use that one. And from my previous comment, ets move this above the `download.file()` chunk to give people context _first_.